README.md

```diff
@@ -33,11 +33,14 @@
 - HTML
 - CSS
 - JavaScript
+- Webpack
 
 
 ### Key Features <a name="Mobile Version of Portfolio"></a>
 
 - **Dynamically add list of tasks**
+- **Check tasks as completed**
+- **Delete checked items**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
![Capture](https://user-images.githubusercontent.com/63087491/215255114-9cb1ac5b-b2d9-41f3-bcd4-265428837b7d.PNG)

- As the previous reviewer mentioned, the procedures to set up your app locally are not sufficient. The app needs to be open using on a browser using webpack dev server(`npm start`) after installing app dependencies using `npm install`